components/components-googledrive/components-googledrive-runtime/src/main/resources/org/talend/components/google/drive/runtime/messages_fr.properties

```
@@ -33,19 +33,19 @@ error.validation.folder.empty=Le nom du dossier ne peut pas \u00EAtre vide.
 error.validation.put.invalid.flow=Flow invalide, v\u00E9rifiez les connecteurs et propri\u00E9t\u00E9s.
 error.validation.local.filename.empty=Le nom du fichier local ne peut \u00EAtre vide.
 error.validation.destinationFolderName.empty=Le nom du dossier de destination ne peut pas \u00EAtre vide.
-error.validation.connection.datastorePath.empty=Le chemin d''acc\u00E8s au DataStore ne peut \u00EAtre vide. D\u00E9finissez-le dans l''onglet Avanc\u00E9 de Connexion.
+error.validation.connection.datastorePath.empty=Le chemin d''acc\u00E8s au Datastore ne peut \u00EAtre vide. Configurez-le dans l'onglet Avanc\u00E9 de Connexion.
 error.validation.connection.proxyHost.empty=L''h\u00F4te du proxy ne peut \u00EAtre vide.
-error.validation.connection.proxyPort.invalid=Le port du proxy est invalide.
+error.validation.connection.proxyPort.invalid=Le port du proxy n''est pas valide.
 error.validation.connection.sslAlgorithm.empty=L'algorithme SSL ne peut \u00EAtre vide.
```

add one more single quote
```suggestion
error.validation.connection.sslAlgorithm.empty=L''algorithme SSL ne peut \u00EAtre vide.
```